com.github.flxzt.rnote.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "com.github.flxzt.rnote",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "rnote",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=session-bus",
```

Comment:
why do you need session bus access? You should just declare which interface you need to access.